.gitignore

```diff
@@ -26,3 +26,6 @@ _testmain.go
 
 coverage.out
 drone-docker
+
+# IDE/Editor related files
+**.swp
```

Comment:
can you revert this file? You can add editor specific files to `.git/info/exclude`